includes/api/betterpayment-payment-request.php

```diff
@@ -7,23 +7,31 @@
 	));
 });
 
-// Only used for Apple Pay
+// Used for Apple Pay and Google Pay express payment methods, because flow is different from usual payment methods
 function payment(WP_REST_Request $request): WP_REST_Response {
 	$url     = Config_Reader::get_api_url() . '/rest/payment';
-	$body    = $request->get_body_params();
-
-	// Add the payment type to the body
-	$body['payment_type'] = 'applepay';
+	$body    = $request->get_json_params();
 
+	// Add the payment type to the body for only pre-defined
```

Comment:
I suggest we raise an error in case neither googlepay_token or applepay_token (is it what's it called) is not present. 

As this would be an exceptional case, a raise of error is appropirate here